### DISCUSSION TITLE
[Feature]: The Plan for the Commercial Font Helvetica Rendering

### DISCUSSION THREAD

**Discussion by jade-zhang-0707** [mozilla/pdf.js#19245] [Q&A]
### Is the feature relevant to the Firefox PDF Viewer?

No

### Feature description

I am currently using PDF.js and noticed the issue regarding the Helvetica font, which is a commercial font. I understand that the use of such a font may involve complex copyright and compliance concerns.
I would like to know if the project has a specific plan to address the presence and handling of the Helvetica font. Are there any intentions to replace it with an alternative font that has a more suitable license for the project? Or are there other strategies in place to ensure compliance while still maintaining compatibility with PDFs that may contain Helvetica text?
It would be greatly appreciated if you could provide some insights or details about the roadmap and considerations for dealing with this particular commercial font. This would help me and other users better understand how PDF.js will manage this aspect in the future and make more informed decisions in our projects.
Thank you for your time and effort in developing and maintaining this valuable project.

**Comment by THausherr** [comment]
Has this been written by an AI?

You "noticed the issue regarding the Helvetica font" - what issue?

**Reply by jade-zhang-0707** [reply]
I am very sorry for the misunderstanding caused by the previous description.
We would like to ask about the use scenarios of commercial font licensing when browsing documentation.
Recently Helvetica font has been commercialized, when using pdf.js for document browsing, if Helvetica is encountered, whether it is the same as the rules authorized before?

**Comment by THausherr** [comment]
Helvetica was always commercial. 
https://en.wikipedia.org/wiki/Helvetica
I don't know about PDF.js (I just hang around here), but applications typically use a font that looks similar when a PDF requires Helvetica and isn't embedding it.

**Reply by jade-zhang-0707** [reply]
Thanks so much for comments.  It is very useful to us.
